scripts/coverage.sh

```diff
@@ -6,8 +6,14 @@ log() {
   echo "$*" >&2
 }
 
-SOLIDITY_COVERAGE=true scripts/test.sh || log "Test run failed"
+# The allFiredEvents file is created inside coverageEnv, but solidity-coverage
+# expects it to be at the top level. We create a symlink to fix this
+ln -s coverageEnv/allFiredEvents allFiredEvents
+
+OZ_TEST_ENV_COVERAGE=true npx solidity-coverage || log "Test run failed"
 
 if [ "$CI" = true ]; then
   curl -s https://codecov.io/bash | bash -s -- -C "$CIRCLE_SHA1"
 fi
+
+rm -f allFiredEvents
```

Comment:
We should add this as a `trap` for `EXIT`. Take a look at the previous test script for reference.

Reply:
Good catch!